lib/features/email/domain/usecases/parse_calendar_event_interactor.dart

```diff
@@ -11,11 +13,26 @@ class ParseCalendarEventInteractor {
 
   ParseCalendarEventInteractor(this._calendarEventRepository);
 
-  Stream<Either<Failure, Success>> execute(AccountId accountId, Set<Id> blobIds) async* {
+  Stream<Either<Failure, Success>> execute(
+    AccountId accountId,
+    Set<Id> blobIds,
+    String emailContents
+  ) async* {
     try {
       yield Right(ParseCalendarEventLoading());
-      final calendarEventList = await _calendarEventRepository.parse(accountId, blobIds);
-      yield Right(ParseCalendarEventSuccess(calendarEventList));
+
+      final result = await Future.wait(
+        [
+          _calendarEventRepository.parse(accountId, blobIds),
```

Comment:
one question, what happen if we can not `parse` the calendar event? How about use the original content?

Reply:
If the calendar content cannot be parsed, we will display the original content of the email (like the old versions).